src/components/HomePage/MoviesSection/MoviesSection.vue

```diff
@@ -1,131 +1,122 @@
 <script>
 import { defineComponent } from 'vue';
 
-import MoviesSectionCard from '@/components/HomePage/MoviesSection/MoviesSectionCard.vue';
-
-const moviesList = [
-{
-id: 1,
-title: "Harry Potter and the Philosopher's Stone",
-genre: { id: 3, name: "Fantasy" },
-poster_url:
-    "https://images-na.ssl-images-amazon.com/images/I/713KEd-8jyL._AC_SL1500_.jpg",
-length: 159
-},
-{
-id: 2,
-title: "Star Wars Episode IV: A New Hope",
-genre: { id: 4, name: "Sci-Fi" },
-poster_url:
-    "https://img01.mgo-images.com/image/thumbnail/v2/content/MMVD85527F13D6BC8CB6518D4B8DF956FAD2.jpeg",
-length: 125
-},
-{
-id: 3,
-title: "You've got Mail",
-genre: { id: 5, name: "Romance" },
-poster_url: "https://miro.medium.com/max/520/0*JeA6QiQ6BzuNzKHP.jpg",
-length: 120
-}
-];
+import MovieCard from '@/components/MoviesPage/MovieCard.vue';
 
 export default defineComponent({
-    name: 'MoviesSection',
-    components: {
-        MoviesSectionCard,
-    },
-    data() {
-        return {
-            movies: moviesList,
+  name: 'MoviesSection',
+  components: {
+    MovieCard
+  },
+  props: {
+    movies: {
+      type: Array,
+      default: [
```

Comment:
Ok, maybe I misinterpreted a suggestion I received during one of the previous CR's.
Was told that I should provide default values on props rather than set them as required.
Thought that it would be nice if the component actually rendered something visual based on the mock data.
But if that's considered bad practice, I'll stop doing it right away!